Resources/public/js/views/tabs/ez-locationviewdetailstabview.js

```diff
@@ -178,16 +179,21 @@ YUI.add('ez-locationviewdetailstabview', function (Y) {
             this.fire('loadUser', {
                 userId: creatorId,
                 attributeName: 'creator',
+                loadingErrorAttributeName: 'creatorLoadingError'
             });
 
             if (creatorId === ownerId) {
                 this.onceAfter('creatorChange', function (e) {
                     this.set('owner', this.get('creator'));
                 });
+                this.onceAfter('creatorLoadingErrorChange', function (e) {
+                    this.set('ownerLoadingError', this.get('creatorLoadingError'));
```

Comment:
Can it be a situation like, a creator is different from an owner?

Reply:
That was #937, @adamwojs?

Reply:
@sunpietro Yes, owner and creator can be different (see steps to reproduce in PR linked by @andrerom). Variables names are misleading, but actually "creator" is the author of the last version ("Last contributor") and "Owner" is the author of the initial version content. For instance: